openscm/utils.py

```diff
@@ -5,8 +5,10 @@
 from typing import Tuple, Union
 import warnings
 
+_ensure_input_is_tuple_calls: int = 0
```

Comment:
why this change?

Reply:
Pylint and Mypy were complaining. Both ways are equally good, I suppose.

Reply:
ok. I have a particular hate for global variables which arose in another project so would prefer to avoid the global if possible. However if this is the only one I'll survive.

Reply:
Totally agree with you on globals ;) I have found a way to avoid that, just a second.

Reply:
Done